resources/views/vendor/cashier/receipt.blade.php

```diff
@@ -0,0 +1,323 @@
+                        <strong>Invoice Number:</strong> {{ $invoiceId }}<br>
+                    @endif
+                </p>
+            </td>
+
+            <!-- Account Name / Header Image -->
+            <td align="right">
+                <span style="font-size: 28px; color: #ccc;">
+                    <strong>{{ $header ?? $vendor ?? $invoice->account_name }}</strong>
+                </span>
+            </td>
+        </tr>
+        <tr valign="top">
+            <td width="50%">
+                <!-- Account Details -->
+                <strong>{{ $vendor ?? $invoice->account_name }}</strong><br>
+
+                @isset($street)
+                    {{ $street }}<br>
+                @endisset
+
+                @isset($location)
+                    {{ $location }}<br>
+                @endisset
+
+                @isset($country)
+                    {{ $country }}<br>
+                @endisset
+
+                @isset($phone)
+                    {{ $phone }}<br>
+                @endisset
+
+                @isset($email)
+                    {{ $email }}<br>
+                @endisset
+
+                @isset($url)
+                    <a href="{{ $url }}">{{ $url }}</a><br>
+                @endisset
+
+                @isset($vendorVat)
+                    {{ $vendorVat }}<br>
+                @else
+                    @foreach ($invoice->accountTaxIds() as $taxId)
+                        {{ $taxId->value }}<br>
+                    @endforeach
+                @endisset
+            </td>
+            <td width="50%">
+                <!-- Customer Details -->
+                <strong>Recipient</strong><br>
+                {{ $invoice->customer_name ?? $invoice->customer_email }}<br>
+                @if ($address = $invoice->customer_address)
+                    @if ($address->line1)
+                        {{ $address->line1 }}<br>
+                    @endif
+
+                    @if ($address->line2)
+                        {{ $address->line2 }}<br>
+                    @endif
+
+                    @if ($address->city)
+                        {{ $address->city }}<br>
+                    @endif
+
+                    @if ($address->state || $address->postal_code)
+                        {{ implode(' ', [$address->state, $address->postal_code]) }}<br>
+                    @endif
+
+                    @if ($address->country)
+                        {{ $address->country }}<br>
+                    @endif
+                @endif
+
+                @if ($invoice->customer_phone)
+                    {{ $invoice->customer_phone }}<br>
+                @endif
+
+                @if ($invoice->customer_name)
+                    {{ $invoice->customer_email }}<br>
+                @endif
+
+                @foreach ($invoice->customerTaxIds() as $taxId)
+                    {{ $taxId->value }}<br>
+                @endforeach
+                
+                {{ $billing }}
```

Comment:
I have a suspicion that strange things happen when someone writes an address with an "&", an "#", an "é" or other special characters ;-) Does multi-line info work?